internal/onelogin/user_info_test.go

```diff
@@ -270,3 +284,93 @@ func TestParseIdentityClaimWhenIdentityPublicKeyFuncError(t *testing.T) {
 	_, err := c.ParseIdentityClaim(context.Background(), UserInfo{})
 	assert.Equal(t, expectedError, err)
 }
+
+func TestCredentialAddressTransformToAddress(t *testing.T) {
```

Comment:
Felt like it was worth duplicating the tests from `place` to ensure the values assigned to `AddressDetails` don't drift at any point